ivy/functional/frontends/tensorflow/activations.py

```diff
@@ -15,6 +15,9 @@ def hard_sigmoid(x):
 def linear(x):
     return x
 
+def relu(x):
+    x = ivy.clip(x, 0.0, ivy.inf)

```

Comment:
I've just checked the docs and there is actually an `ivy.relu()` function in the [API](https://lets-unify.ai/ivy/functional/ivy/activations/relu/relu_functional.html?highlight=relu#ivy-relu), you can just use it instead :)